lcx-api/adapter.js

```diff
@@ -0,0 +1,71 @@
+const { Requester, Validator } = require('@chainlink/external-adapter')
+
+const customError = (data) => {
+  return !data || !data.data || !data.data.Price
+}
+
+const customParams = {
+  symbol: false
+}
+
+const headers = {
+  'Content-Type': 'application/json',
+  'api-key': process.env.API_KEY
+}
+
+const ALL = 'all'
+const SYMBOLS = ['btc-usd', 'btc-eur', 'eth-usd', 'eth-eur']
+
+function getUrlForSymbol (symbol) {
+  if (symbol === ALL) {
+    return SYMBOLS.map(i => getUrlForSymbol(i))
+  } else {
+    const s = symbol.split('-')
+    return `https://rp.lcx.com/v1/rates/current/?coin=${s[0].toUpperCase()}&currency=${s[1].toUpperCase()}`
+  }
+}
+
+function retrieveRates (url) {
+  return Requester.request({
+    url,
+    headers
+  }, customError)
+}
```

Comment:
Please also use an arrow function here to keep it consistent.

```suggestion
const requestRates = (url) =>
  Requester.request({
    url,
    headers
  }, customError)
```